{"stats": {"errors": 1, "failures": 0, "passes": 0, "skipped": 0, "encoding": "UTF-8", "total": 1}, "results": [{"classname": "index_test:FieldElementTest", "name": "test_div", "time": 0.0004639625549316406, "type": "error", "errtype": "builtins.ValueError", "message": "pow() 2nd argument cannot be negative when 3rd argument specified", "tb": "Traceback (most recent call last):\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/conda/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n  File \"/home/pb-tester-ccdle12/ecc-div-jsong-pb-sf-072618/test/index_test.py\", line 11, in test_div\n    self.assertEqual(a**-3, FieldElement(29, 31))\n  File \"/home/pb-tester-ccdle12/ecc-div-jsong-pb-sf-072618/index.ipynb\", line 71, in __pow__\n    \"print(17 * pow(3, prime-2, prime) % prime)\\n\",\nValueError: pow() 2nd argument cannot be negative when 3rd argument specified\n"}]}